cleansweep/forms.py

```diff
@@ -115,7 +115,7 @@ class SendMailForm(Form):
                     ('contacts', 'All Contacts')
                 ])
     subject = StringField('Subject', validators=[validators.Required()])
-    message = TextAreaField("Message", validators=[validators.Required()])
+    message = TextAreaField("Message", id="email", validators=[validators.Required()])
```

Comment:
please specify the id when rending the field. I think this is not the right place.